src/components/Toggle/Toggle.tsx

```diff
@@ -4,12 +4,12 @@ import React from 'react';
 
 import { ComponentPropsWithoutRef } from 'react';
 
-export interface ToggleProps extends ComponentPropsWithoutRef<'div'> {
-  size?: 'sm' | 'md' | 'lg';
+export interface ToggleProps extends ComponentPropsWithoutRef<'input'> {
+  tShirtSize?: 'sm' | 'md' | 'lg' | undefined;
```

Comment:
`size` is a number is `ComponentPropsWithoutRef<'input'>` so we need to rename it